Client/Frontend/Browser/Tabs/InactiveTabItemCell.swift

```diff
@@ -0,0 +1,142 @@
+        containerView.addSubviews(bottomSeparatorView)
+        containerView.addSubview(leftImageView)
+        containerView.addSubview(midView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        bottomSeparatorView.translatesAutoresizingMaskIntoConstraints = false
+        leftImageView.translatesAutoresizingMaskIntoConstraints = false
+        midView.translatesAutoresizingMaskIntoConstraints = false
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+
+        containerViewLeadingConstraint = containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor)
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            containerViewLeadingConstraint,
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+
+            leftImageView.heightAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.widthAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                   constant: InactiveTabItemCellModel.UX.ImageViewLeadingConstant),
+            leftImageView.centerYAnchor.constraint(equalTo: midView.centerYAnchor),
+            leftImageView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor,
+                                               constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+            leftImageView.bottomAnchor.constraint(lessThanOrEqualTo: containerView.bottomAnchor,
+                                                  constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+
+            midView.topAnchor.constraint(equalTo: containerView.topAnchor),
+            midView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            midView.leadingAnchor.constraint(equalTo: leftImageView.trailingAnchor, constant: 13),
+            midView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -7),
+
+            titleLabel.topAnchor.constraint(equalTo: midView.topAnchor,
+                                            constant: InactiveTabItemCellModel.UX.LabelTopBottomMargin),
+            titleLabel.bottomAnchor.constraint(equalTo: midView.bottomAnchor,
+                                               constant: -InactiveTabItemCellModel.UX.LabelTopBottomMargin),
+            titleLabel.leadingAnchor.constraint(equalTo: midView.leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: midView.trailingAnchor),
+
+            bottomSeparatorView.heightAnchor.constraint(equalToConstant: 0.7),
```

Comment:
0.7 seems strange as I've seen in other places we had 1 for line height. Can we confirm this? And also, I would maybe put this in the UX struct.